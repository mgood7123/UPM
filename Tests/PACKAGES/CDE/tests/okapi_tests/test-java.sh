#!/bin/sh

source okapi_test_common.sh
okapi_test_init # call init function

$OKAPI_BIN /usr/bin/java "" $TESTDIR

pushd $TESTDIR > /dev/null
find . | xargs file | sort > contents.txt
popd > /dev/null

diff -u $TESTDIR/contents.txt $TESTNAME.golden
