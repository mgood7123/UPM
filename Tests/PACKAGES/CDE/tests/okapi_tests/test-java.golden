./contents.txt:                                        empty
.:                                                     directory
./etc/alternatives:                                    directory
./etc/alternatives/java:                               symbolic link to `../../usr/lib/jvm/jre-1.6.0-openjdk/bin/java'
./etc:                                                 directory
./usr/bin:                                             directory
./usr/bin/java:                                        symbolic link to `../../etc/alternatives/java'
./usr:                                                 directory
./usr/lib:                                             directory
./usr/lib/jvm:                                         directory
./usr/lib/jvm/java-1.6.0-openjdk-1.6.0.0:              directory
./usr/lib/jvm/java-1.6.0-openjdk-1.6.0.0/jre/bin:      directory
./usr/lib/jvm/java-1.6.0-openjdk-1.6.0.0/jre/bin/java: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), dynamically linked (uses shared libs), for GNU/Linux 2.6.9, stripped
./usr/lib/jvm/java-1.6.0-openjdk-1.6.0.0/jre:          directory
./usr/lib/jvm/jre-1.6.0-openjdk:                       symbolic link to `java-1.6.0-openjdk-1.6.0.0/jre'
